library/std/src/sys/cmath/mod.rs

```diff
@@ -0,0 +1,11 @@
+#![cfg(not(test))]
+
+cfg_if::cfg_if! {
+    if #[cfg(target_os = "windows")] {
+        mod windows;
+        pub use windows::*;
```

Comment:
Should we keep platform specific things in pal?

Reply:
If they implement a shared interface, I'd say no, so that similar code stays together.

Reply:
Ok, that makes sense to me as a principle.